src/styles/utils/variable.scss

```diff
@@ -0,0 +1,27 @@
+// Nav
+$nav-hight: 60px;
```

Comment:
The variable name `$nav-hight` seems to have a typo. Consider renaming it to `$nav-height` to reflect the correct spelling.